src/components/ha-picture-upload.ts

```diff
@@ -39,13 +49,32 @@ export class HaPictureUpload extends LitElement {
 
   public render(): TemplateResult {
     if (!this.value) {
+      /* eslint-disable lit-a11y/anchor-is-valid */
```

Comment:
I am not too sure about this check. Due I think to recent changes to dialogs/navigation, the href=# didn't work anymore, so I had to change to href="javascript:".

Eslint however doesn't seem to like the new "fake" anchor. I'm not sure how else to open a dialog from a link like was requested though.

Reply:
Mhh yeah not the nicest solution.

What about using a `button` tag and style it like a link?